tmt/steps/scripts/tmt-reboot-core

```diff
@@ -19,7 +19,12 @@ read -r test_pid test_reboot_file < "$TMT_TEST_PIDFILE"
 
 mkdir -p "$(dirname "$test_reboot_file")"
 
-printf '{"command": "%s", "timeout": "%s"}' "$command" "$timeout" > "$test_reboot_file"
+# Build reboot-request JSON, including only non-empty command/timeout to satisfy
+# strict Pydantic validation.
+jq -n --arg cmd "$command" --argjson timeout "${timeout:-null}" '
```

Comment:
This would require `jq` to be predemt on every guest, every distro, every architecture. I am afraid you need to stick to bash.